### PR TITLE
Reference GNU Guix installation

### PR DESCRIPTION
**Pull request by ghost**
Transient was added to GNU Guix, and can be installed that way for people who are not fond of pip.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,10 @@ Installation
 ------------
 
 `transient` is available on [PyPI](https://pypi.org/project/transient/), so the latest
-release can be installed with `pip install transient`. To install `transient` from
+release can be installed with `pip install transient`. It is also available from
+[GNU Guix](http://guix.gnu.org/), and can be installed with `guix install python-transient`.
+
+To install `transient` from
```
Comment (ALSchwalm) [inline README.md]: Looks like the formatting got a little messed up here, can you fix that?

**Comment by ALSchwalm** [comment]
Thanks for the PR! I'm glad to know that transient has been packaged for Guix

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: To be clear, you mean the line break on 35 and 36? I'd be happy to remove it if that is the case.

Comment (ALSchwalm) [reply README.md]: Yep, just that line break (but please try to keep things to an approximately 80 character line limit). Also, it's probably worth making this same change to the docs at: https://github.com/ALSchwalm/transient/blob/master/docs/installation.md. That way, if someone is just looking at the readthedocs page, they'll still see the guix option

**Comment by ghost** [comment]
> Thanks for the PR! I'm glad to know that transient has been packaged for Guix

We should be all square now!

**Comment by ALSchwalm** [comment]
Looks good to me. Merged, and thanks again for the contribution

**Pull request merged**